<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TariffRateWidget</class>
 <widget class="QWidget" name="TariffRateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="tarrifRate3">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Добровольное медицинское страхование:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="tarrifRate1">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Установите тарифную ставку</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Страхование домашнего имущества:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Страхование автотранспорта:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
font: 500 14pt &quot;Montserrat Medium&quot;;
    background-color: #4CAF50; 
    color: white; 
    border: none;
    font-size: 14px;
    border-radius: 5px; 
}
QPushButton:hover {
    background-color: #45a049;
}
QPushButton:pressed {
    background-color: #357a38; 
}</string>
         </property>
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="tarrifRate2">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
